### PR TITLE
[TL] Beach Wars! & Endless Quest Description

### PR DESCRIPTION
**Pull request by efhunter**
Translated description for "Beach Wars!" and the Endless Quest category.

### CODE REVIEW

**Comment by efhunter** [comment]
lol oops, just realized this is the old beach wars and not 2018, time to translate that as well real quick

File: Quests/Descriptions/quest_00050150.csv [hunk 1]
```diff
@@ -1,5 +1,5 @@
 Title#0,"""Beach Wars!"""
-Explanation#0,"""夏だっ！\u3000海だっ！\u3000シーおねえさんだっ！<br>惑星ウォパルの海岸はまさに今が旬っ！<br>なのに暑さのせいか、凶暴化した海王種や<br>突然現れるようになったダーカーやらで<br>もー、バカンス計画がむっちゃくちゃ！<br>憩いの場を取り戻すためにも<br>アークスのみんなの力を貸して欲しいなっ！<br>さあ、まばゆいビーチを取り戻そうっ！<br>――――――――――――――――――――<br>Fail: Time limit expires"""
+Explanation#0,"""Summer's here! The beach is here! And big sis Xie is here too! Now is THE season for beach-going on Wopal! Yet, perhaps because of the heat, the Oceanids have gone wild, and Darkers have suddenly appeared, leaving our vacation plans in ruins! We need all ARKS to help take our oasis of relaxation back! So, come on, let's get this beautiful beach back!<br>――――――――――――――――――――<br>Fail: Time limit expires"""
```
Comment (SynthSy) [inline Quests/Descriptions/quest_00050150.csv]: These descriptions have text limits and do require linebreaks, but we have to do this. It's why they weren't translated.

File: Quests/Seasonal/quest_00050880.csv [hunk 2]
```diff
@@ -1,8 +1,8 @@
 Title#0,"""Beach Wars 2018!"""
-Explanation#0,"""惑星ウォパルの海岸エリアにて<br>突如としてダーカー、エスカダーカー<br>更には海王種の出現が確認された。<br>またマザー・クラスタの使徒に酷似した<br>幻創種の出現も報告されている。<br>出現地は保養地にも近く、看過はできない。<br>アークス各員は、急ぎ現地へと向かい<br>状況の調査と鎮圧にあたれ。<br>――――――――――――――――――――<br>失敗条件/制限時間の経過"""
-NpcComMsg1#0,"""これより、転送を開始します。<br>今回の任務は、ビーチに巣くう<br>エネミーたちの掃討です。"""
-NpcComMsg2#0,"""アラトロンさんに似た姿のエネミーも<br>確認されています。みなさんで協力して<br>先へ進んでください！"""
-NpcComMsg3#0,"""ほかにも謎の反応が確認されています。<br>ナス、ナカ……？\u3000よくわかりませんが<br>見つけたら、助けてあげてください！"""
+Explanation#0,"""A sudden appearance of Darkers, ESC-A Darkers, as well as Oceanids in Wopal's coast area has been confirmed. There are also reports of Phantoms with a strong resemblance to Apostles from the Mother Cluster. The appearances are close to a relaxation area, and cannot be overlooked. All ARKS, head to the target location at once, and begin your investigation and suppression.<br>――――――――――――――――――――<br>Failure condition: Time limit expires"""
```
Comment (SynthSy) [inline Quests/Seasonal/quest_00050880.csv]: see above

File: Quests/Seasonal/quest_00170400.csv [hunk 3]
```diff
@@ -1,8 +1,8 @@
 Title#0,"""T:Beach Wars 2018!"""
-Explanation#0,"""惑星ウォパルの海岸エリアにて<br>突如としてダーカー、エスカダーカー<br>更には海王種の出現が確認された。<br>またマザー・クラスタの使徒に酷似した<br>幻創種の出現も報告されている。<br>出現地は保養地にも近く、看過はできない。<br>アークス各員は、急ぎ現地へと向かい<br>状況の調査と鎮圧にあたれ。<br>――――――――――――――――――――<br>失敗条件/制限時間の経過"""
-NpcComMsg1#0,"""これより、転送を開始します。<br>今回の任務は、ビーチに巣くう<br>エネミーたちの掃討です。"""
-NpcComMsg2#0,"""アラトロンさんに似た姿のエネミーも<br>確認されています。みなさんで協力して<br>先へ進んでください！"""
-NpcComMsg3#0,"""ほかにも謎の反応が確認されています。<br>ナス、ナカ……？\u3000よくわかりませんが<br>見つけたら、助けてあげてください！"""
+Explanation#0,"""A sudden appearance of Darkers, ESC-A Darkers, as well as Oceanids in Wopal's coast area has been confirmed. There are also reports of Phantoms with a strong resemblance to Apostles from the Mother Cluster. The appearances are close to a relaxation area, and cannot be overlooked. All ARKS, head to the target location at once, and begin your investigation and suppression.<br>――――――――――――――――――――<br>Failure condition: Time limit expires"""
```
Comment (SynthSy) [inline Quests/Seasonal/quest_00170400.csv]: See above

File: Quests/Descriptions/quest_00050150.csv (hunk 1 above)
Comment (efhunter) [reply Quests/Descriptions/quest_00050150.csv]: I see. In that case, should I just revert the descriptions and leave the NPC text?

Comment (SynthSy) [reply Quests/Descriptions/quest_00050150.csv]: We'll see able adding your translations in, but they won't be an exact match.

**Comment by SynthSy** [comment]
Now in your branch, edit `PSO2.en_NPC.dict` in the _aspell folder, and add `Nasu` as the last item in the list.

**Comment by alama** [comment]
Sorry, updated your branch/PR to update in line of PSO2's update

**Pull request merged**